ocs_ci/ocs/ui/page_objects/buckets_tab.py

```diff
@@ -1,18 +1,146 @@
 import json
-
+import logging
+import uuid
 import requests
+import random
+import string
+import os
+import time
+
 from selenium.webdriver.common.by import By
+from selenium.common.exceptions import NoSuchElementException
+
 from ocs_ci.ocs.ocp import get_ocp_url
 from ocs_ci.ocs import exceptions
 from ocs_ci.ocs.ui.page_objects.confirm_dialog import ConfirmDialog
-from ocs_ci.ocs.ui.page_objects.object_storage import ObjectStorage, logger
+from ocs_ci.ocs.ui.page_objects.object_storage import ObjectStorage
 from ocs_ci.utility import version
 
+logger = logging.getLogger(__name__)
+
 
 class BucketsTab(ObjectStorage, ConfirmDialog):
+    """
+    A class representation for abstraction of Buckets tab related OpenShift UI actions
+    """
+
     def __init__(self):
         super().__init__()
```

Comment:
since we do serious changes here, can you pls along this remove `super().__init__()` from the constructor? it is unnecessary and takes time to instantiate parents up to BaseUI.